appeals/functions/document-processing/document-move/blob.js

```diff
@@ -9,7 +9,11 @@ const storageClient = BlobStorageClient.fromUrl(config.BO_BLOB_STORAGE_ACCOUNT);
  * @param {string} destinationUrl
  */
 export const copyBlob = async (sourceUrl, destinationUrl) => {
-	const blobPath = new URL(destinationUrl).pathname.replace(`/${config.BO_BLOB_CONTAINER}/`, '');
+	const destinationComponents = destinationUrl.split(`/${config.BO_BLOB_CONTAINER}/`);
```

Comment:
Why do you need regex here? Why not just `destinationUrl.split(config.BO_BLOB_CONTAINER);` ?

Reply:
not a regex, just building a string part of the URL `/blobcontainer/`

Reply:
need to take `http://myurl/mybasepath/${config.BO_BLOB_CONTAINER}/...` and extract `...`

Reply:
ah sorry, long day 😅

Reply:
This isn't regex, its a template string (adding `/` either side)